<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>781</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="grid_layout"/>
   </widget>
   <widget class="QPushButton" name="dark_mode_toggle_btn">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menunew_game">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="open_file_menu"/>
    <addaction name="new_game_menu"/>
    <addaction name="actionsolve"/>
   </widget>
   <addaction name="menunew_game"/>
  </widget>
  <action name="open_file_menu">
   <property name="text">
    <string>open file</string>
   </property>
  </action>
  <action name="new_game_menu">
   <property name="text">
    <string>new game</string>
   </property>
  </action>
  <action name="actionsolve">
   <property name="text">
    <string>solve</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
